src/v/pandaproxy/schema_registry/handlers.cc

```diff
@@ -407,7 +407,14 @@ post_subject(server::request_t rq, server::reply_t rp) {
     auto inc_del{
       parse::query_param<std::optional<include_deleted>>(*rq.req, "deleted")
         .value_or(include_deleted::no)};
-    vlog(plog.debug, "post_subject subject='{}', deleted='{}'", sub, inc_del);
```

Comment:
We only handle the normalize flag for json schemas for now. Should we error out on `normalize=false` for non-json requests?

Reply:
> We only handle the normalize flag for json schemas for now. Should we error out on `normalize=false` for non-json requests?

`false` is the default. For avro, it's always true, regardless of whether it's asked for or not. For protobuf, it's complicated/not implemented, but rejecting anything would be a regression.